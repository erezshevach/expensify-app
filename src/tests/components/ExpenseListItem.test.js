import React from "react";
import { shallow } from "enzyme";
import ExpenseListItem from "../../components/ExpenseListItem";
import expensesList from "../fixtures/expensesList";

test('should render ExpenseListItem with expense details', () => {
    const wrapper = shallow(<ExpenseListItem expenses={expensesList[0]}/>);
    expect(wrapper).toMatchSnapshot();
});
