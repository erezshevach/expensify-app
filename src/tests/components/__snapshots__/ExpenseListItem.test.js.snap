// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem with expense details 1`] = `
<div>
  <Link
    to="/edit/undefined"
  >
    <h3 />
  </Link>
  <p>
     - 
  </p>
</div>
`;
